tests/legacy-distro-packaging.nix

```diff
@@ -0,0 +1,71 @@
+{ pkgs, rosenpass-deb, rosenpass-rpm }:
+
+let
+  wg-deb = pkgs.fetchurl {
+    url = "https://ftp.de.debian.org/debian/pool/main/w/wireguard/wireguard-tools_1.0.20210914-1.1_amd64.deb";
+    hash = "sha256-s/hCUisQLR19kEbV6d8JXzzTAWUPM+NV0APgHizRGA4=";
+  };
+  wg-rpm = pkgs.fetchurl {
+    url = "https://mirrors.n-ix.net/fedora/linux/releases/40/Everything/x86_64/os/Packages/w/wireguard-tools-1.0.20210914-6.fc40.x86_64.rpm";
```

Comment:
Does this need to be tracked and updated from time to time?

Reply:
I think the link will work for a very long time.

The necessity to upgrade the rpm and deb packages might possibly come up whenever the distros are updated.

Upgrading would look like this:

1.) One would update the `nix-vm-test` input and re-run the tests.
2.) In the worst case, the packages wouldn't work in the tests any longer for whatever reason
3.) The updater would put updated URLs and hashes in the fetcher calls.

The whole process would take a few minutes and be performed max once a year if not less often - i wouldn't advise to invest the time for any automation here.